jetson/nav/gate_search/gateStateMachine.cpp

```diff
@@ -114,13 +119,82 @@ NavState GateStateMachine::executeGateSpin()
             return NavState::GateTurn;
         }
         nextStop += waitStepSize;
-        return NavState::GateSpinWait;
+        return NavState::GateWait;
     }
     return NavState::GateSpin;
 } // executeGateSpin()
 
+//Executes the logic for a gimbal gate search. The main objective of a gimbal gate search is to spin the gimbal
+//to positive "gimbalSearchAngleMag" (150) then to -150 then to 0. Every "wait step size" we stop the gimbal
```

Comment:
Should we change this step size since our range is [-179.5,88.5]? Or would it not matter?

Reply:
Yea I should edit that comment. waitStepSize is now deprecated since it doesn't make sense for bounds that are "weird" like this.